__tests__/multidatespicker.test.js

```diff
@@ -0,0 +1,746 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const $ = require('jquery');
+require('jquery-ui/ui/widgets/datepicker');
+
+if (!$.ui) {
+  $.ui = {};
+}
+
+if (!$.ui.multiDatesPicker) {
+  $.ui.multiDatesPicker = { version: '1.6.6' };
+}
+
+require('../jquery-ui.multidatespicker.js');
```

Comment:
This is not needed. Any needed initialization is already speficied in `jest.setup.js`